index.html

```diff
@@ -4,9 +4,11 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="./CSS/styles.css/"">
+    <link rel="stylesheet" href="./CSS/styles.css" />
   </head>
 
   <body></body>
-  <header></header>
+  <header>
```

Comment:
Header should be put in body

Reply:
Hallo Laura! Danke für den Kommentar. Das ist eigentlich garnicht meine aktuelle Version. Ich weiß auch nicht genau, was da los ist.

Reply:
Ach nee. Tatsächlich. Du hattest Recht! Danke.